cmd/okctl/venv.go

```diff
@@ -70,6 +78,15 @@ func buildVenvCommand(o *okctl.Okctl) *cobra.Command { //nolint: funlen
 		),
 	}
 
+	flags := cmd.Flags()
+
+	flags.StringVarP(&opts.ClusterDeclarationPath,
```

Comment:
Is there any way we can avoid dealing with a global flag in venv.go? upgrade.go doesn't seem to need it.

Reply:
this is not globale? opts is local to buildVenvCommand?

Reply:
As discussed: global flag as in "-c" used for all commands.

Reply:
added a `addClusterDeclarationPathFlag()` function to call for all subcommands that need it